### PR TITLE
 Make `docusign-client` package buildable with stack. 

### PR DESCRIPTION
**Pull request by jonathanknowles**

### CODE REVIEW

File: src/DocuSign/Client/Types.hs [hunk 1]
```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE DeriveDataTypeable #-}
```
Comment (hasufell) [inline src/DocuSign/Client/Types.hs:1]: Can you elaborate? Why is this needed for **stack**?

File: stack.yaml [hunk 2]
```diff
@@ -1,3 +1,8 @@
 resolver: lts-11.13
 packages:
 - .
+- location:
```
Comment (hasufell) [inline stack.yaml:4]: As I will upload a hackage release, this won't be needed anymore.

File: src/DocuSign/Client/Types.hs (hunk 1 above)
Comment (jonathanknowles) [reply src/DocuSign/Client/Types.hs:1]: Ah, my bad! Looks like my PR is not rebased correctly. I'll make a new PR.
